docs/source/deployment/airflow_astronomer.md

```diff
@@ -2,161 +2,153 @@
+```shell
+root:
+  handlers: [console]
+```
+
+5. Package the Kedro pipeline as a Python package so you can install it into the Airflow container later on:
 
 ```shell
 kedro package
 ```
 
 This step should produce a wheel file called `new_kedro_project-0.1-py3-none-any.whl` located at `dist/`.
 
-* **Step 2.2**: Add the `src/` directory to `.dockerignore`, as it's not necessary to bundle the entire code base with the container once we have the packaged wheel file.
+6. Convert the Kedro pipeline into an Airflow DAG with `kedro airflow`
 
 ```shell
-echo "src/" >> .dockerignore
+kedro airflow create --target-dir=dags/ --env=airflow
 ```
 
-* **Step 2.3**: Modify the `Dockerfile` to have the following content:
+This step should produce a .py file called `new_kedro_project_dag.py` located at `dags/`.
+
+### Deployment process with Astro Airflow
+
+In this section, we'll start by setting up a new blank Airflow project using Astro. We'll then copy the files prepared in the previous section from our Kedro project. Next, we'll customize the Dockerfile to enhance logging capabilities and manage the installation of our Kedro package. Finally, we will run and explore the Airflow cluster.
+
+1. [Initialise an Airflow project with Astro](https://docs.astronomer.io/astro/cli/develop-project) in a new folder outside of your Kedro project. Let's call it `kedro-airflow-spaceflights`
+
+    ```shell
+    cd ..
+    mkdir kedro-airflow-spaceflights
+    cd kedro-airflow-spaceflights
+    astro dev init
+    ```
+
+2. The folder `kedro-airflow-spaceflights` will be executed within the Airflow container. To run our Kedro project there, we need to copy several items from the previous section into it:
+- the `/data` folder from Step 1, containing sample input datasets for our pipeline. This folder will also store the output results.
+- the `/conf` folder from Steps 2-4, which includes our DataCatalog, Parameters, and Logging customisation files. These files will be used by Kedro during its execution in the Airflow container.
+- the `.whl` file from Step 5, which we will install in the Airflow Docker container to execute our project node by node.
+- the Airflow DAG from Step 6 for deployment in the Airflow cluster.
+    ```shell
+    cd ..
+    cp -r new-kedro-project/data kedro-airflow-spaceflights/data
+    cp -r new-kedro-project/conf kedro-airflow-spaceflights/conf
+    mkdir -p kedro-airflow-spaceflights/dist/
+    cp new-kedro-project/dist/new_kedro_project-0.1-py3-none-any.whl kedro-airflow-spaceflights/dist/
+    cp new-kedro-project/dags/new_kedro_project_dag.py kedro-airflow-spaceflights/dags/
+    ```
+
+3. Add a few lines to the `Dockerfile` to set the environment variable `KEDRO_LOGGING_CONFIG` to point to `conf/logging.yml` to enable custom logging in Kedro and to install the .whl file of our prepared Kedro project into the Airflow container:
```

Comment:
You should specify which `Dockerfile` you mean, because at this point we've created `new-kedro-project` and `kedro-airflow-spaceflights`

Reply:
Agree, fixed